### ISSUE TITLE
It doesn't work very well with this PDF.

### ISSUE THREAD

**Issue by datermine** [Nutlope/pdftochat#3]
Here's an example of a PDF it doesn't work well at all with:
https://nysirestakes.com/backend/News/news_upload/2023_Breeders_Award_12123_1706.pdf

Sample prompt: What are the headers of the table?

**Comment by Nutlope** [comment]
I appreciate you reporting this! Yeah I don't think it does too well with tables to be honest since I pass it all in as just text. Perhaps a feature to implement, which is detecting tables and embedding them in a certain format

**Comment by ajaxbo360** [comment]
yeah its a feature to implement , detecting tables should be nice to have .

**Comment by rudro12356** [comment]
I also tried this with a research paper and it didn't work well. The pdf had tables, charts and texts. The model seemed to be hallucinating.

**Comment by hynra** [comment]
How about using a custom document loader like [Unstructred](https://unstructured.io/)? Unstructred is also available on Langchain.
